From: Manoj Srivastava <srivasta@debian.org>
Date: Mon, 20 Mar 2006 15:37:15 -0600
Subject: Remove documentation from build system

The doc directory is removed from the source tarball by mk-origtargz, so
delete all reference to it in the build system.

Forwarded: not-needed
Last-Update: 2023-02-12
---
 Makefile.am  | 4 +---
 configure.ac | 2 +-
 2 files changed, 2 insertions(+), 4 deletions(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -21,13 +21,11 @@
 
 MAKE_HOST =	@MAKE_HOST@
 
-SUBDIRS =	lib po doc
+SUBDIRS =	lib po
 
 bin_PROGRAMS =	make
 include_HEADERS = src/gnumake.h
 
-man_MANS =	doc/make.1
-
 make_SRCS =	src/ar.c src/arscan.c src/commands.c src/commands.h \
 		src/debug.h src/default.c src/dep.h src/dir.c src/expand.c \
 		src/file.c src/filedef.h src/function.c src/getopt.c \
--- a/configure.ac
+++ b/configure.ac
@@ -543,7 +543,7 @@
 
 # Specify what files are to be created.
 AC_CONFIG_FILES([build.cfg tests/config-flags.pm \
-                 Makefile lib/Makefile doc/Makefile po/Makefile.in])
+                 Makefile lib/Makefile po/Makefile.in])
 # We don't need this: the standard automake output suffices for POSIX systems.
 #mk/Posix.mk
 
